spot-ml/src/main/scala/org/apache/spot/utilities/data/InputOutputDataHandler.scala

```diff
@@ -0,0 +1,50 @@
+package org.apache.spot.utilities.data
+
+import org.apache.log4j.Logger
+import org.apache.hadoop.fs.{LocatedFileStatus, Path, RemoteIterator, FileUtil => fileUtil}
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{DataFrame, SQLContext}
+
+
+/**
```

Comment:
fixed but not showing as changed because of line 9 wasn't actually modified but what's beneath that line. Please see full changes to find updated version.